tests/Issues/issue450Test.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Issues;
+
+use Mockery;
+use Mpdf\Color\ColorConverter;
+use Mpdf\CssManager;
+use Mpdf\Image\ImageProcessor;
+use Mpdf\Image\Svg;
+use Mpdf\Language\LanguageToFont;
+use Mpdf\Language\ScriptToLanguage;
+use Mpdf\Otl;
+use Mpdf\SizeConverter;
+
+/**
+ * Class Issue450Test
+ * @author Antonio Norman - softcodex.ch
+ */
+class Issue450Test extends \Mpdf\BaseMpdfTest
+{
+
+	/**
+	 * SVGs with global CSS styles, had the styles ignored
+	 */
+	public function testSvgWithGlobalStyles()
+	{
+		// Copied from  Mpdf\Image\MetricsGeneratorTest
+		$otl = Mockery::mock(Otl::class);
+		$cssManager = Mockery::mock(CssManager::class);
+		$imageProcessor = Mockery::mock(ImageProcessor::class);
+		$this->sizeConverter = $this->createMock(SizeConverter::class); // Mockery::mock(ColorConverter::class); <-- doesn't work for me !??!
```

Comment:
How are these comments relevant? Either find out why Mockery doesn't work and fix it, or do not comment at all.

Reply:
Or state the reason of not working.